### DISCUSSION TITLE
Accuracy for archeo-astronomy

### DISCUSSION THREAD

**Discussion by decoomanj** [Stellarium/stellarium#3120] [General]
Yesterday I had a discussion with an academic researcher who claimed that Stellarium cannot be used for archeo-astronomical purposes because the calculations are not accurate enough if you go back in time. He claimed that even for medieval times, Stellarium would be off, not to mention bronze-age times. I was surprised by this claim since quite some people are using Stellarium exactly for these purposes. Unfortunately, I do not have the possibility to compare the accuracy of Stellarium to other software. Since that person who claimed this, just came up with "I've talked to an astronomer and he said this and that" as a source, I wanted to check with the Stellarium community if this is a false or true claim. (I'm well aware that the fault on calculations will be higher the further you go back in time, but that's also the case for others, right?)

**Comment by gzotti** [comment]
Around 2010 this academic researcher was right. As an academic researcher myself, I also saw many problems. However, the open-source nature of this project allows inspecting and fixing them. 

Since V0.14.2 Stellarium has state of the art precession/nutation. The User Guide has several more milestones listed in the preface, I am getting tired of listing them. The dropping of the zero version number had a reason. Appendix F in the User Guide discusses accuracy of planetary positions against JPL Horizon. Apart from some issues around stellar proper motion which are likewise mentioned in Appendix F, I would like to receive comparison data which would show "computed with Stellarium" vs. "Computed with &lt;whatever reference data can be used in the Mesolithic&gt;" and a hint of what may be wrong. For the time being, we don't compute diurnal aberration, and likewise no light deflection in gravitational fields of Sun or Jupiter. Likewise, minor planets and comets move on classical unperturbed Kepler ellipses. And some moons of the outer planets run on simple Keplerian ellipses with orbital elements of unclear origin. This should not pose a problem for naked-eye observer in the Bronze age, though.

I only discuss the desktop version, of course.

**Reply by SpaceHighlander** [reply]
Hi I'm new - sorry if this is wrong place (I was searching forum to find the answer to the similar question) 

*my problem >
when I want to check Equinox in 3000bc and set the date to -3000 March 20th the SUN is Not 'due East' ! - why ?
- how the Stellarium Date (Calendar) works ? 
*I understand precession and shift of the stars etc. but why 'equinox' (SUN) is not on the 'correct' DATE ?

**Reply by gzotti** [reply]
Why do you assume equinox in 3000BC should be on March 20 on the (proleptic) Julian Calendar? This is a strange but widespread misunderstanding about our common historical calendars. It has nothing to do with Stellarium.

You check equinox in 3000BC in Stellarium by going to astronomical year -2999 and set the date where the Sun crosses the celestial equator. This, and not some named date in a human-made list, is the astronomical definition of equinox. Then you may see some date displayed in the calendar panel, or, with the Calendars plugin activated, in a few dozen other calendar systems which were also not in use in 3000BC. In 3000BC nobody called the equinox date "March 22" or "April 15", and the predecessors of the Maya likely still not 0.5.15.4.15, so whatever the calendar date panel shows is just a name in modern calendars that were introduced more than halfway between then and today.

**Reply by SpaceHighlander** [reply]
yes I understand but I thought it's sync with the precession ?
- in case of spring equinox in year -3000 the SUN rise due East approx 3 weeks later (then should) - which mean the 'correct date' was around 10-12 April (in Stellarium calendar) 
- so what is the point of the DATE/calendar if it deosn't work ?
*NOTE that in this case I can 'see' that the SUN position is wrong and I can 'correct' the date but if this is wrong anybody who check other random 'DATES' will use 'wrong dates' but without knowing it !

**Reply by gzotti** [reply]
What does precession have to do with the calendar? Please, while reading Appendix F, read Appendix F.8 thoroughly. Equinoxes and solstices are defined by the Sun, Earth, the ecliptic, and not much more. No arbitrary human action or counting involved. (Although the definition is of course human...)

In 3000BC sun rises due east at spring equinox. Also in 2000BC, 1000BC or any other year. (Neglecting refraction, of course.)
Please take your time machine and ask Julius Caesar why he made such bad calendar in the mid-1st century BC that does not even stretch 3000 years back in time. But be considerate - compare his regular 4-years leap rule to the calendar mess the Romans used before!
Calendars are man-made constructions that try to synchronize civil activities with natural cycles. Most calendars don't work highly accurately for more than a few centuries after their introduction. 
Our common school knowledge of "20./21. March == Spring Equinox" is the result of the ecclesiastic requirement to repair the procedures of finding the dates of Easter that seemingly did no longer match the Nicean definitions from the early 4th century. This reform process itself took centuries! The Gregorian calendar, introduced in the Catholic world in or not much after 1582, but not in use in the British sphere before the mid-18th century, does a very good job, but also only for a few thousand years. And we (at least the "globalized" world) are living with this practical calendar for centuries now, taking it as perennial truth.

In lack of a better calendar, and given a good millennium of dated texts and chronological events before the reform, historians commonly use the Julian for everything before 1582, or may even specify their own calendar rules for prehistoric dates in introductory chapters of their books (making them incompatible to other authors' dates!). Or they may just refer to "10 days after spring equinox" some plants started to blossom or such. Astronomers can compute and date astronomical events described in Cuneiform texts, providing absolute reference dates in otherwise relative chronology. These dates are then commonly written in the Julian calendar. Other calendars for other past periods and cultures are usually known by a few experts only. 

You can read a date in the Proleptic Gregorian Calendar (i.e., dated way before its introduction) using the Calendars plugin. If you write this in a text, you must explicit specify in which calendar this date is to be understood.

You can also simply specify equinox 3000BC as JD625777.5, but, while accurate and unambiguous, it's even less intuitive.

**Reply by SpaceHighlander** [reply]
thank you for the info but my point is ::
- 99% users of Stellarium use the DATE window to check the SKY at some specific TIME - but this is NOT the actual DATE and the SKY (stars etc) > avarage user is not astronomer who can compute 'correct date'

so my point is : using Stellarium DATE window for checking the 'correct' SKY at specific date is incorrect !
*without computation is generally useless to check correct SKY at specific date & time 

*the Equinox & 3000bc was only example - yes it doesn't matter it can be 2000bc -- but using the DATE window and the date 20/21 March is not correct --- equinox example it's obvious because you can see that is incorrect
- but most people who want to check ANY date don't actually see correct SKY and they don't know it !

**again I thought the computation is automatic or synchronized and by using DATE window I see correct SKY 
- now I know I don't

***Yes the JD system isn't intuitive and I don't know how to use it for 'correct' date

**Reply by gzotti** [reply]
The date is correct as given in the respective calendar. It is the calendar that has errors keeping seasons and dates rigorously linked. I don't know how much more I must write when everything can be deduced from the first paragraph of Appendix F.8.   In 3000BC there was no month called "March", and therefore you must take caution with what you actually want to see.  To set equinox sky, USE THE ASTRONOMICAL DEFINITION and disregard the human-given name/number of the day! To set the sky for March 20 in the Julian calendar, you can do that. Just be aware this was not equinox day. Please read Wikipedia or other material on calendars, there is enough on this simple topic out there. It is not a software problem. Stellarium as astronomically accurate tool just illustrates what you set.

**Reply by SpaceHighlander** [reply]
I understand the equinox and the fact the calendars are not perfect (and why) 
I did get the answer to my 1st question 

the rest is just conclusion that avarage user (not pro astronomer) don't realize that - and they are using DATE window to check the SKY at specific date but this is incorrect method ! (i'm not talking about equinox but any date) 

but lets move on 
if I don't know how to calculate 'correct' date can I use 'manual' method ?
- 1st checking 'correct' date for equinox -- aligning the SUN due East and use the number of days as reference ?
-  for eample : if in 1860 bc the equinox is 10days later this mean if I want to check the 'correct' sky at some specific day in this year I should shift +10 days ?

**Reply by gzotti** [reply]
If interested users don't read the manual and don't know or understand how historians work, yes, they will set some date and will get the correct sky for that date as defined in the Julian or Gregorian calendar, and not necessarily what they wanted. If you carve a statue and cut yourself in the finger, don't blame the cutler - the knife has worked.

The equinox of 3000BC is the moment in the astronomical year -2999 when the Sun crosses the equator. 
The equinox of 1860BC is the moment in the astronomical year -1859 when the Sun crosses the equator. 

You can use Stellarium to find any of these days. As is generally known, the month/day in the Julian calendar will not be the same, which however is historically irrelevant. 

The day 10 days after after equinox is best called "10 days after equinox".

**Comment by decoomanj** [comment]
Thank you for your quick response. So, if I understand it correctly, when I turn back time to for example -3000, the observation of stars should be almost perfect, right? The diurnal aberration wouldn't play a noticeable role in checking the position of the stars?

**Reply by gzotti** [reply]
Planets: should be "perfect". Stars: not sure, a handful of bright stars (and many non-naked eye stars) may be somewhat off. For stars you should check with CdC. 
Diurnal aberration is sub-arcseconds, nothing to bother IMHO.
What you cannot get right, principally, is &Delta;T much before 720BC, as there are no trustworthy records available to track this irregularly growing offset. Early-morning Solar eclipses in the Neolithic in Newgrange? NO WAY!

**Reply by decoomanj** [reply]
Ok, but ΔT shouldn't be a blocking issue if I want to check the rising of a given star on a particular date in -3000? We can all live with an error margin for those dates.

**Reply by gzotti** [reply]
As said everywhere, for STAR POSITIONS you should expect some larger errors, and currently should use Cartes du Ciel for reference.

**Comment by Carrickview** [comment]
The oldest framework for timekeeping and annual reckoning is based on the transition of the stars from an evening to dawn appearance as they change position from left to right of the Sun in response to the Earth's orbital motion-

https://sol24.net/data/html/SOHO/C3/96H/VIDEO/

".. on account of the procession of the rising of Sirius by one day in the course of 4 years,.. therefore it shall be, that the year of 360 days and the 5 days added to their end, so one day shall be from this day after every 4 years added to the 5 epagomenae before the new year" Canopus Decree 238 BC

The satellite tracking along with the Earth through space with its focus on our central star allows contemporary observers to appreciate what a heliacal rising is in its raw state and minus any influence from daily rotation, which swamps observations on the surface of a planet that also rotates daily.

The Ptolemaic framework is an extension of this core framework but uses the motion of the Sun through the background stars for predictive purposes-

http://astro.dur.ac.uk/~ams/users/sun_ecliptic.gif

"Moreover, we see the other five planets also retrograde at times and
stationary at either end [of the regression]. And whereas the sun
always advances along its own direct path, they wander in various
ways, sometimes straying to the south and sometimes to the north; that
is why they are called "planets" [wanderers]. Copernicus

This framework allows for predictions within the dates of the 365/366 calendar system but is further removed from the older framework, and goodness knows how old the heliacal rising framework is.

http://astro.dur.ac.uk/~ams/users/solar_year.gif

The last framework is RA/Dec, and much software is built on this scheme which allows observers to not only predict solar system events within the calendar framework but also to introduce clocks to give the exact times within a 24-hour day and date. It is the basis of the clockwork solar system. This framework is even further removed from the heliacal framework and the system of observations employed by the first Sun-centred astronomers.

I will work with anyone who will streamline observations and make the major adjustments necessary for this era with satellite imaging. Projecting the Earth's daily rotation into space as a celestial sphere framework has very limited uses, whereas the actual change in the position of the stars relative to the Sun and its glare is more productive for many uses. Finally, it is the actual motion of the stars within the structure of the galaxy that offers observers a more comprehensive view of our solar system and galaxy.

As an aside, Mercury has recently transitioned from a dawn to twilight appearance as it moves behind the central Sun from right to left as demonstrated by the C3 camera on the tracking satellite.
